HISTORY.md

```diff
@@ -11,6 +11,9 @@
 * Add support for trace sampling.
 * Enable properties block checksum verification for block-based tables.
 
+### Public API Change
+* Remove 'ttl' param from 'CompactionOptionsFIFO'. The option has been deprecated and 'ttl' in 'AdvancedColumnFamilyOptions' is used instead.
```

Comment:
- There's already a "Public API Change" section below. So lets just append to that. 
- Also since `ColumnFamilyOptions` inherits `AdvancedColumnFamilyOptions`, lets just say `ColumnFamilyOptions` here  as that is how all the RocksDB users interact:  s/AdvancedColumnFamilyOptions/ColumnFamilyOptions/